app.js

```diff
@@ -57,7 +57,7 @@ app.get('/feature-list', function(req, res) {
     db.any('SELECT DISTINCT name FROM feature').then(data => {
         res.json(data.map(f => f.name));
     }).catch(err => {
-        res.json("[]");
+        res.status(404).send({client: "No features found"});
```

Comment:
I was wondering what you'd change, this makes more sense than what I was planning lol. And thanks for using my modal response format! 💯